### DISCUSSION TITLE
The fuction "Copy to " can't copy to another album.

### DISCUSSION THREAD

**Discussion by Mowen-M** [LycheeOrg/Lychee#2662] [Bugs] [answered]
### Lychee version

V6.0

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

The fuction "Copy to " can't copy to another album. It only copy to subfolder of the picture. I am using Docker to build up it.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Mowen-M** [comment]
And also, could you change the search character to two character? Because most Chinese word are two like 
![image](https://github.com/user-attachments/assets/16e547bc-9517-47e2-8566-a2c1c5c071fa)
Or what should i change in the config to change it?

**Reply by ildyria** [reply]
Settings => All settings => Mod Search => Number of characters required to trigger search (default: 4).

**Comment by Mowen-M** [comment]
The share link overview doesn't work.
![image](https://github.com/user-attachments/assets/17935948-9ba0-43d3-9d7a-6353d65c7fbc)

**Reply by d7415** [reply]
That shows existing sharing. You can add new sharing permissions from individual albums.

Adding from there may be a useful enhancement.

**Reply by ildyria** [reply]
planned but no time. :sweat_smile:

**Comment by ildyria** [comment, marked as answer]
Fixed https://github.com/LycheeOrg/Lychee/pull/2655 .
